frontend/src/features/maps/routes/MapCreateForm.tsx

```diff
@@ -130,9 +130,9 @@ export default function MapCreateForm() {
         rings: [
           [
             { x: 0.0, y: 0.0 },
-            { x: 0.0, y: 10_000.0 },
-            { x: 10_000.0, y: 10_000.0 },
-            { x: 10_000.0, y: 0.0 },
+            { x: 0.0, y: 1_000.0 },
```

Comment:
This is now 10x10m if I'm not mistaken. The 100x100m square as a default would take to long to calculate with granularity 10cm (at least it would take to long if everybody had that set as a default).
Since a default of 10x10 is more performant (and probably more realistic for most people anyways) I reduced it.

Reply:
That is too small, even for our garden. Our garden has 200m length.

If there are unsolvable performance issues, please change the granularity of the blocks to 1m.